CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Docker v3.11.3_7.5.1
+
+### Added
+
+- Update to Wazuh version 3.11.3_7.5.1
```

Comment:
Please change the Elastic version to `7.5.2`.

Reply:
Fixed on latest commit